### PR TITLE
Count component: Add primary prop and style

### PR DESCRIPTION
**Pull request by sirreal**
Add prop `primary` to `Count` component.
Add `is-primary` styling.

**To test:**
* https://calypso.live/devdocs/design/count?branch=update/count-component-primary
* Ensure styling is appropriate.
* Ensure there are no console errors or warning.
* Check usage in #15488 

**Screenshot:**

Original left, with `primary` prop right.

primary

Included in #15488

via @rickybanister in https://github.com/Automattic/wp-calypso/pull/15427#issuecomment-310759356

### CODE REVIEW

**Comment by folletto** [comment]
Can you add before/after screenshots? Thanks!

**Comment by rickybanister** [comment]
Nice change, this makes more sense and is more sustainable.

**Comment by sirreal** [comment]
> Can you add before/after screenshots? Thanks!

@folletto Yup, added to issue.

**Comment by folletto** [comment]
Ok! As Rick also confirmed, I'll remove the Design Review tag. Design wise 👍

File: client/components/count/docs/example.jsx [hunk 1]
```diff
@@ -8,7 +8,10 @@ import React from 'react';
 import Count from 'components/count';
 
 const count = () => (
-	<Count count={ 65365 } />
+	<div>
+		<Count count={ 65365 } />
+		<Count primary count={ 65366 } />
```
Comment (tyxla) [inline client/components/count/docs/example.jsx]: How about adding the `primary` prop to the component README as well?

Comment (sirreal) [inline client/components/count/docs/example.jsx]: Nice addition as usual 🙂 
303f9db5e05feee8cbafdd3369e214a21b8b54e9

**Pull request merged**